docs/ROADMAP.md

```diff
@@ -0,0 +1,119 @@
+## Q3 Roadmap - Tentative
+
+### Overview
+
+---
+
+In Q3, there are a few groups of tasks we'd like to work on:
+
+- **Continue with the stories from Q2**: Authentication, Authorization,
+  Inclusion of Related models
+- **Feature parity gap**: Declarative support
+- **Internal tooling/infrastructure**: support Node.js 12 in juggler &
+  connectors, reduce build time, etc.
+- **Juggler-next**: some groundwork to prepare us to work on juggler-next, e.g.
+  spike on how the code resides affect our build process
+
+### Scope
+
+---
+
+#### Authentication
+
+- [ ] Token based authentication in API Explorer.
+      [Spike](https://github.com/strongloop/loopback-next/issues/2027)
+- [ ] Stretch goal: enable class level `@authenticate` decorator:
+      https://github.com/strongloop/loopback-next/issues/2460
+
+#### Authorization
+
+- [ ] Complete the `Add authorization component` PR:
+      https://github.com/strongloop/loopback.io/pull/857
+- [ ] Common layer for authentication & authorization
+      https://github.com/strongloop/loopback-next/issues/2900
+- [ ] Add authorization to example-shopping repo
+
+#### Inclusion of Related Models https://github.com/strongloop/loopback-next/issues/1352
+
+- awaiting to @bajtos' spike to create the tasks
+
+#### From model definition to REST API with no custom repository/controller classes (Declarative Support) https://github.com/strongloop/loopback-next/issues/2036
+
+- [ ] [CRUDRESTController](https://github.com/strongloop/loopback-next/issues/2736)
+- [ ] [KeyValueRESTController](https://github.com/strongloop/loopback-next/issues/2737)
+- [ ] [Spike: Booter for creating REST APIs from model files](https://github.com/strongloop/loopback-next/issues/2738)
+
+#### CI cleanup https://github.com/strongloop-internal/scrum-apex/issues/422
+
+- [ ] Fix CI for
+      [dashdb](https://github.com/strongloop/loopback-connector-dashdb/issues/76)
+- [ ] Fix CI for
+      [db2](https://github.com/strongloop/loopback-connector-db2/issues/130)
+      connector
+- [ ] Stretch goal: Test juggler 3.x and 4.x on connectors
+
```

Comment:
I noticed that postgresql connector is failing on cis-jenkins Node.js 10, can we add a task to fix it?

Reply:
added https://github.com/strongloop/loopback-connector-postgresql/issues/384